observer-blindata-server/src/main/java/org/opendatamesh/platform/up/metaservice/blindata/schema_analyzers/utils/InternalRefResolver.java

```diff
@@ -0,0 +1,54 @@
+package org.opendatamesh.platform.up.metaservice.blindata.schema_analyzers.utils;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public abstract class InternalRefResolver {
```

Comment:
Added an utility class to resolve internal references. External references should be resolved prior by the ODM registry